src/aws/dynamodb.utils.ts

```diff
@@ -28,3 +30,7 @@ export function getLeaderboardKey(chain: Chain): string {
 export function getChainStartBlockKey(chain: Chain, block: number): string {
   return `${chain.network}_${block}`;
 }
+
+export function getVaultEntityId({ network }: Chainish, { address }: Vaultish): string {
```

Comment:
we should start to transition to using this for `id` for all vault entities, which currently we base off only address